extensions/extensions-docker/aissemble-model-training-api-containers/aissemble-model-training-api-sagemaker/src/main/resources/docker/Dockerfile

```diff
@@ -1,37 +1,43 @@
 # Script for creating base AIOps Model Training API image
-
 ARG DOCKER_BASELINE_REPO_ID
 ARG VERSION_AISSEMBLE
-FROM ${DOCKER_BASELINE_REPO_ID}boozallen/aissemble-fastapi:${VERSION_AISSEMBLE} AS builder
 
-RUN python -m venv /opt/venv
-ENV PATH="/opt/venv/bin:$PATH"
+#HABUSHU_BUILDER_STAGE - HABUSHU GENERATED CODE (DO NOT MODIFY)
```

Comment:
Q: although this will be replaced each build, we always want to check in the updated Dockerfile and that's why this is included?

Reply:
Yeah I think we treat it just like we do the `pyproject.toml` file changes that Habushu makes.  They are fairly stable and should only change when the underlying logic changes.